<launch>
    <node pkg="action_estimator" type="action_estimator.py" name="action_estimator" output="screen" ns="action">
        <!-- topic name to subscribe to -->
        <param name="img_topic" type="str" value="/yolov7/image" />
        <!-- topic name to subscribe to -->
        <param name="pose_topic" type="str" value="/yolov7/kpt" />
        <!-- topic name for the detection output -->
        <param name="out_topic" type="str" value="action_est" />
        <!-- queue size for publishing -->
        <param name="queue_size" type="int" value="10" />
        <!-- flag whether to also publish image with the visualized detections -->
        <param name="visualize" type="bool" value="false" />
    </node>
</launch>